cache-config/t3c-apply/t3c-apply.go

```diff
@@ -146,6 +146,19 @@ func Main() int {
 	}
 
 	if cfg.UseGit == config.UseGitYes || cfg.UseGit == config.UseGitAuto {
+		//need to see if there is an old lock file laying around.
+		const gitMaxLockAge = 5
```

Comment:
can you comment on the time units or put them in the variable name?  Might be a pet peeve of mine.

Reply:
Added comment and time units to variable name.